src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -51,7 +53,6 @@ export default class CardanoSignTransaction extends AbstractMethod {
             { name: 'ttl', type: 'amount' },
             { name: 'certificates', type: 'array', allowEmpty: true },
             { name: 'withdrawals', type: 'array', allowEmpty: true },
-            { name: 'metadata', type: 'string' },
             { name: 'validityIntervalStart', type: 'amount' },
```

Comment:
I'd rather throw an error if metadata is accidentally passed so existing integrations do not stop working correctly without the developers noticing

Reply:
Done. https://github.com/vacuumlabs/connect/pull/12/commits/62ab5cd64855bbb00fca699641fc8d3c8069d853#diff-3f63cf5f20614bf66b007cac352a997338d942f06ddec324b3eae61b8e1f3548R49 I've also added a test for that just in case. https://github.com/vacuumlabs/connect/pull/12/commits/62ab5cd64855bbb00fca699641fc8d3c8069d853#diff-95d2cba6d3ec639fa89710a8e43e188282fa57da8eae165242e5d20392417f93R497